services/postgresflex/wait/wait_test.go

```diff
@@ -300,6 +299,53 @@ func TestDeleteInstanceWaitHandler(t *testing.T) {
 	}
 }
 
+func TestForceDeleteInstanceWaitHandler(t *testing.T) {
+	tests := []struct {
+		desc             string
+		instanceGetFails bool
+		instanceState    string
+		wantErr          bool
+	}{
+		{
+			desc:             "delete_succeeded",
+			instanceGetFails: false,
+			instanceState:    InstanceStateSuccess,
```

Comment:
Noticed now, shouldn't this be?

```suggestion
			instanceState:    InstanceStateDeleted,
```